lib/platforms.nix

```diff
@@ -8,8 +8,9 @@ rec {
   openbsd = ["i686-openbsd" "x86_64-openbsd"];
   netbsd = ["i686-netbsd" "x86_64-netbsd"];
   cygwin = ["i686-cygwin" "x86_64-cygwin"];
-  unix = linux ++ darwin ++ freebsd ++ openbsd;
-  all = linux ++ darwin ++ cygwin ++ freebsd ++ openbsd;
+  illumos = ["x86_64-solaris"];
```

Comment:
Hm, it's tricky - I don't have a "real" solaris machine at my disposal... and do we even want to support it?